frontend/app/src/components/assets/AssetValueRow.vue

```diff
@@ -4,13 +4,15 @@ import { useAggregatedBalancesStore } from '@/store/balances/aggregated';
 import { type AssetPriceInfo } from '@/types/prices';
 
 const props = defineProps({
-  identifier: { required: true, type: String }
+  identifier: { required: true, type: String },
+  isCollectionParent: { required: false, type: Boolean, default: false }
 });
-const { identifier } = toRefs(props);
+
+const { identifier, isCollectionParent } = toRefs(props);
 const { assetPriceInfo } = useAggregatedBalancesStore();
 
 const info = computed<AssetPriceInfo>(() => {
-  return get(assetPriceInfo(identifier));
+  return get(assetPriceInfo(identifier, get(isCollectionParent)));
```

Comment:
`assetPriceInfo` returns a computed, do you think we can pass a MaybeRef param unref inside and get rid of the extra computed around?